src/components/BookForm.js

```diff
@@ -1,25 +1,56 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { useDispatch } from 'react-redux';
-import { addBook } from '../redux/books/booksSlice';
+import { addBook, removeBook } from '../redux/books/booksSlice';
+import api from './Api';
 import Button from './Button';
```

Comment:
- Hi, I went through the requirement of the project and find out that you need to **refactor your addBook and removeBook functions with createAsyncThunk**. Therefore, I would advise you to follow the requirements and import `createAsyncThunk`
- Open these 2 links to know more about [createAsyncThunk](https://dev.to/ifeanyichima/what-is-createasyncthunk-in-redux--mhe) / [createAsyncThunk](https://redux-toolkit.js.org/api/createAsyncThunk)